scripts/jest/visual-adapter.js

```diff
@@ -1,7 +1,18 @@
 // @flow
-const { toMatchImageSnapshot } = require('jest-image-snapshot');
 
-expect.extend({ toMatchImageSnapshot });
+// const kebabCase = require('lodash/kebabCase');
+const { configureToMatchImageSnapshot } = require('jest-image-snapshot');
+
+expect.extend({
```

Comment:
Changes here are for a new PR after this has been approved to shorten the snapshot names

Reply:
`customSnapshotIdentifier` specifically needed to be set for `jest.retryTimes` to work